types/taiko/index.d.ts

```diff
@@ -0,0 +1,366 @@
+// Custom Typings for Taiko - https://taiko.gauge.org/#/
+// Custom type definitions for Taiko 1.0.5
+//
+
+declare module 'taiko' {
+    export type BrowserEvent =
+        | 'DOMContentLoaded'
+        | 'loadEventFired'
+        | 'networkAlmostIdle'
+        | 'networkIdle'
+        | 'firstPaint'
+        | 'firstContentfulPaint'
+        | 'firstMeaningfulPaint';
+
+    export interface BrowserOptions {
+        headless?: boolean;
+        args?: string[];
+        host?: string;
+        port?: number;
+        ignoreCertificateErrors?: boolean;
+        observe?: boolean;
+        observeTime?: number;
+        dumpio?: boolean;
+    }
+
+    export interface EventOptions {
+        waitForEvents?: BrowserEvent[];
+    }
+
+    export interface BasicNavigationOptions {
+        waitForNavigation?: boolean;
+        navigationTimeout?: number;
+    }
+
+    export interface NavigationOptions extends BasicNavigationOptions, EventOptions {
+        headers?: Map<string, string>;
+        waitForStart?: boolean;
+    }
+
+    export interface ClickOptions extends NavigationOptions {
+        button?: 'left' | 'right' | 'middle';
+        clickCount?: number;
+        elementsToMatch?: number;
+    }
+
+    export interface GlobalConfigurationOptions extends BasicNavigationOptions {
+        observeTime?: number;
+        retryInterval?: number;
+        retryTimeout?: number;
+    }
+
+    export interface TapOptions extends BasicNavigationOptions, EventOptions {}
+
+    export interface KeyOptions extends NavigationOptions {
+        text?: string;
+        delay?: number;
+    }
+
+    export interface WriteOptions extends NavigationOptions {
```

Comment:
extends eventOptions too

Reply:
hmm, 
```typescript
NavigationOptions extends BasicNavigationOptions, EventOptions
and
WriteOptions extends NavigationOptions
```
I think that the extension is transitive and it should already be included...